setup.py

```diff
@@ -127,10 +127,12 @@ def run(self):
         'Intended Audience :: Developers',
         'Natural Language :: English',
         'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3 :: Only',
```

Comment:
What do these two lines mean? Is `3` just any version of v3, or the latest? And what does the "Only" mean?